react/features/conference/components/Conference.native.js

```diff
@@ -192,12 +193,17 @@ class Conference extends Component<Props> {
                 onClick = { this._onClick }
                 style = { styles.conference }
                 touchFeedback = { false }>
+                <StatusBar
+                    hidden = { true }
+                    translucent = { true } />
 
                 {/*
                   * The LargeVideo is the lowermost stacking layer.
                   */}
                 <LargeVideo />
 
+                <ConferenceNotification />
```

Comment:
Web has a concept of a notification. Do the concepts of ConferenceNotification and the web notification overlap enough to be grouped in the same directory?

Reply:
Hmm, this is a good question, and frankly, I haven't thought about it. I just checked the code, and based on the reducer, it can actually work... @jitsi/mobile what do you think about this?